### PR TITLE
translate edge component documentation

### PR DESCRIPTION
**Pull request by zjcomeon**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:
translate edge component documentation

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/README_zh.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!--
+ * @Author: zj.wang
+ * @Date: 2021-12-22 10:59:00
+ * @LastEditors: zj.wang
+ * @LastEditTime: 2021-12-22 11:56:26
+ * @Description: 
+-->
```
Comment (gy95) [inline edge/README_zh.md:7]: remove this block content.

Comment (zjcomeon) [reply edge/README_zh.md]: ok

Comment (gy95) [reply edge/README_zh.md]: LGTM, and could you please use `git rebase` to squash the commits to only one? that's more clearer.

Comment (zjcomeon) [reply edge/README_zh.md]: ok

Comment (gy95) [reply edge/README_zh.md]: LGTM /assign @fisherxu

**Review by fisherxu — COMMENTED** [review]
Because the main lauguage in community is English, so the Chinese version always outdated because of lack sync.. I think that's enough to have the English version? We also a chinese version of this in root README.md.

**Pull request closed without merging**